tests/test_inlinekeyboardmarkup.py

```diff
@@ -86,13 +86,18 @@ def test_from_column(self):
         assert len(inline_keyboard_markup[1]) == 1
 
     def test_expected_values(self, inline_keyboard_markup):
-        assert inline_keyboard_markup.inline_keyboard == self.inline_keyboard
+        assert inline_keyboard_markup.inline_keyboard == tuple(
```

Comment:
why not just make `self.inline_keyboard` a tuple of tuples?

Reply:
b/c I want to make sure that on init a list of list in converted into a tuple of tuples